packages/indy-vdr/src/anoncreds/IndyVdrAnonCredsRegistry.ts

```diff
@@ -218,20 +219,26 @@ export class IndyVdrAnonCredsRegistry implements AnonCredsRegistry {
 
       const response = await pool.submitReadRequest(request)
 
+      //FIXME: Is the ledgerType 1 ok or is it meant to be a dynamic value?
+      // if yes, what the best way to get the ledgerType
+      const schema = await this.fetchIndySchemaWithSeqNo(agentContext, response.result.ref, did, 1)
+
       if (response.result.data) {
-        return {
-          credentialDefinitionId: credentialDefinitionId,
-          credentialDefinition: {
-            issuerId: didFromCredentialDefinitionId(credentialDefinitionId),
-            schemaId: response.result.ref.toString(),
-            tag: response.result.tag,
-            type: 'CL',
-            value: response.result.data,
-          },
-          credentialDefinitionMetadata: {
-            didIndyNamespace: pool.indyNamespace,
-          },
-          resolutionMetadata: {},
+        if (schema) {
```

Comment:
I think we can write it as: `if (response.result.data && schema)` so we can remove the nesting of the if statements